config/secrets.yml

```diff
@@ -14,7 +14,7 @@ development:
   secret_key_base: <%= ENV["SECRET_KEY_BASE"] %>
 
 test:
-  secret_key_base: <%= ENV["SECRET_KEY_BASE"] %>
+  secret_key_base: 7053b205c9559f83960f18d23f10d36cacd737c0e17e0dd71cbb90d2f3e6
```

Comment:
@mfojtik FWIW I added a hard coded value based on this doc:
http://guides.rubyonrails.org/upgrading_ruby_on_rails.html#config-secrets-yml

IIUC, it suggests generating a value with `rake secret` and using that for development and test, while production gets a value from the environment.